refinery/core/views.py

```diff
@@ -974,7 +974,7 @@ class of this backend as the sender.
                                      request=request)
         return new_user
 
-    def get_success_url(self, request, user):
+    def get_success_url(self, user):
```

Comment:
Did the signature change between versions? Would a test that breaks under the old but works with the new be appropriate?

Reply:
We upgraded registration_redux from 1.1 to 1.9...the signature changed.